src/__configuration__/themes/mid-autumn-festival.ts

```diff
@@ -1,6 +1,6 @@
-import * as PXBThemes from '@pxblue/react-themes';
-import * as Colors from '@pxblue/colors';
-import * as BrandingColors from '@pxblue/colors-branding';
+import * as BluiThemes from '@brightlayer-ui/react-themes';
```

Comment:
https://github.com/search?q=org%3Apxblue+BluiThemes&type=code
We have used `BLUIThemes` instead of BluiThemes. Could you please change?